glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -556,6 +557,25 @@ open class GleanInternalAPI internal constructor () {
     internal fun testHasPingType(pingName: String): Boolean {
         return LibGleanFFI.INSTANCE.glean_test_has_ping_type(handle, pingName).toBoolean()
     }
+
+    @VisibleForTesting
+    internal var isMainProcess: Boolean? = null
+
+    /**
+     *  Returns true if we are running in the main process false otherwise.
+     */
+    private fun Context.isMainProcess(): Boolean {
```

Comment:
Do we really need to extend `Context` with this, since we're the only ones using it, once?

Reply:
Nope, we can change it to a regular function.  Extensions don't have any additional overhead, that I can see, in case that's a concern, and doing it inside of the GleanInternalApi class limits the scope of the extension, so it really just changes it from a statically called function to a non-statically accessed one.